property_tests/arbitraries/stable_b_tree_map_arb.ts

```diff
@@ -6,33 +6,76 @@ import { CandidValueAndMetaArb } from './candid/candid_value_and_meta_arb';
 export const StableBTreeMapArb = fc
     .tuple(
         CandidValueAndMetaArb(),
+        argumentInfoArb(),
         CandidValueAndMetaArb(),
+        argumentInfoArb(),
         UniqueIdentifierArb('stableBTreeMap'),
         createUniquePrimitiveArb(
             fc.nat({
                 max: 254
             })
         )
     )
-    .map(([keySample, valueSample, uniqueIdentifier, memoryId]) => {
-        const name = `stableBTreeMap${uniqueIdentifier}`;
-
-        const imports = new Set([
-            ...keySample.src.imports,
-            ...valueSample.src.imports,
-            'stableJson',
-            'StableBTreeMap'
-        ]);
-
-        return {
-            name,
-            imports,
-            definition: `let ${name} = StableBTreeMap<${keySample.src.candidTypeAnnotation}, ${valueSample.src.candidTypeAnnotation}>(${memoryId});`,
+    .map(
+        ([
             keySample,
-            valueSample
-        };
-    });
+            keyArgumentInfo,
+            valueSample,
+            valueArgumentInfo,
+            uniqueIdentifier,
+            memoryId
+        ]) => {
+            const name = `stableBTreeMap${uniqueIdentifier}`;
+
+            const imports = new Set([
+                ...keySample.src.imports,
+                ...valueSample.src.imports,
+                'stableJson',
+                'StableBTreeMap'
+            ]);
+
+            const serializableArguments = getSerializableArguments(
+                keySample,
+                valueSample,
+                keyArgumentInfo,
+                valueArgumentInfo
+            );
+
+            return {
+                name,
+                imports,
+                definition: `let ${name} = StableBTreeMap<${keySample.src.candidTypeAnnotation}, ${valueSample.src.candidTypeAnnotation}>(${memoryId}${serializableArguments});`,
+                keySample,
+                valueSample
+            };
+        }
+    );
 
 // TODO there must be a better way to get this type out, the sample will actually run
 export const StableBTreeMap = fc.sample(StableBTreeMapArb)[0];
 export type StableBTreeMap = typeof StableBTreeMap;
+
+function getSerializableArguments(
+    keySample: StableBTreeMap['keySample'],
+    valueSample: StableBTreeMap['valueSample'],
+    keyArgumentInfo: 'STABLE_JSON' | 'CANDID_TYPE_OBJECT',
+    valueArgumentInfo: 'STABLE_JSON' | 'CANDID_TYPE_OBJECT'
```

Comment:
Seems like this union type should be pulled out rather than duplicated.

Also, `keyArgumentInfo` and `valueArgumentInfo` don't feel informative to me. These seem to be serialization formats. Maybe consider a name more like that.